test_suite.py

```diff
@@ -64,153 +64,222 @@ def run(self, command):
 
 @pytest.fixture
 def docker_runner(docker_client, docker_image):
-    runner = DockerRunner(docker_client, docker_image)
+    runners = []
+    def _docker_runner(language):
+        runner = DockerRunner(docker_client, docker_image(language))
+        runners.append(runner)
+        return runner
+
+    yield _docker_runner
+
+    # I don't love having a list to append to, but don't know how to find the runner
+    # in order to do clean up after the yield
```

Comment:
Yeah, so making the `runners` list feels  so there is a way to grab the created container feels wonky...